docker_db_config_template/8.29.6/EXAConf

```diff
@@ -0,0 +1,229 @@
+        Group = exausers
+        LoginEnabled = False
+        AdditionalGroups = exaadm, exadbadm, exastoradm, exabfsadm, exasaasadm
+    [[exasaasusr]]
+        ID = 499
+        Group = exausers
+        LoginEnabled = False
+        AdditionalGroups = exasaasadm,
+
+[Node : 11]
+    PrivateNet = {{ private_network }}
+    PublicNet =
+    Name = n11
+    # Affinity decides how this node is used in the cluster. i.e, the possibility to become master
+    Affinity = 11
+    UUID = F0EA64D47F374A00B0530772981C559EE1C59086
+    DockerVolume = n11
+    [[Disk : disk1]]
+        Component = exastorage
+        Devices = dev.1
+        Mapping = dev.1:/exa/data/storage
+
+# Global EXAStorage options
+[EXAStorage]
+    # Enable or disable background recovery / data restoration (does not affect on-demand recovery)
+    BgRecEnabled = True
+    # Max. throughput for background recovery / data restoration (in MiB/s)
+    BgRecLimit =
+    # Space usage threshold (in percent, per node) for sending a warning
+    SpaceWarnThreshold = 90
+
+# An EXAStorage volume
+[EXAVolume : DataVolume1]
+    # Type of volume: 'data' | 'archive'
+    Type = data
+    # Volume size (e. g. '1 TiB')
+    Size = {{ disk_size }}
+    # Name of the disk to be used for this volume.
+    # This disk must exist on all volume nodes.
+    Disk = disk1
+    # Comma-separated list of node IDs for this volume (put dedicated redundancy nodes at the end, if any)
+    Nodes = 11
+    # OPTIONAL: Nr. of master nodes for this volume. Remaining nodes will be used for redundancy only.
+    NumMasterNodes = 1
+    # Desired redundancy for this volume
+    Redundancy = 1
+    BlockSize = 4 KiB
+    StripeSize = 256 KiB
+    # User and group IDs that own this volume (e. g. '1000:1005')
+    Owner = 500 : 500
+    Permissions = rwx
+    # OPTIONAL: shared volumes can be opened (for writing) by multiple clients simultaneously
+    Shared = True
+    # OPTIONAL: I/O priority (0 = highest, 20 = lowest)
+    Priority = 10
+
+# RemoteLogging : None/None
+[Logging]
+    LogRotationTypes = local,
+    RemoteLogRotationVolume = None
+    RemoteLogRotationPrefix = None
+
+# An EXASOL database
+[DB : DB1]
+    # Unique id of this database.
+    ID = aCbUZhU3R8eEc0UIDS7wQw
+    # Version nr. of this database.
+    Version = {{ db_version }}
+    # Memory size over all nodes (e. g. '1 TiB').
+    MemSize = {{ mem_size }}
+    Port = {{ db_port }}
+    # User and group IDs that own this database (e. g. '1000:1005').
+    Owner = 500 : 500
+    # Comma-separated list of node IDs for this DB (put reserve nodes at the end, if any).
+    Nodes = 11
+    # Nr. of initially active nodes for this DB. The remaining nodes will be reserve nodes.
+    NumActiveNodes = 1
+    # Name of the data volume to be used by this database.
+    DataVolume = DataVolume1
+    EnableAuditing = True
+    BuiltinScriptLanguageName = slc-6.0.0-c4-5-standard-EXASOL-8.0.0/ScriptLanguages-standard-EXASOL-8.0.0-slc-v6.0.0-PB5EHDLN
```

Comment:
this causing the problem

Reply:
This needs to be removed. 
`BuiltinScriptLanguageName` was moved to the global section (See line 43 in this file). 

However, COS implements an upgrade logic, so that it overwrites the value from `Db` to the global section.  -> As we have here the old SLC (which is not part of the Docker image) it fails to find the SLC then.